recipes/genenotebook/meta.yaml

```diff
@@ -13,17 +13,17 @@ source:
 requirements:
   host:
     - jq
-    - nodejs=12.14.1
+    - nodejs=14.17.1
     - python=2.7
   build:
     - {{ compiler('cxx') }}
   run:
     - blast
-    - nodejs=12.14.1
-    - mongodb>=4.0
+    - nodejs=14.17.1
+    - mongodb=4.0.3
```

Comment:
Was there a particular reason to pin to this exact version? There's a 5.0.3 available

Reply:
Yes, currently genenotebook is only compatible with nodejs 14.x and mongodb 4.x, for various reasons. A future release will go to nodejs 16.x and mongodb 5.x.